src/modal-template.ts

```diff
@@ -3,9 +3,9 @@ import { property, customElement } from 'lit/decorators.js';
 
 import '@internetarchive/ia-activity-indicator/ia-activity-indicator';
 import '@internetarchive/icon-close';
-import iaIcon from '@internetarchive/icon-ia-logo/index.js';
```

Comment:
I have changed the way to render the ia-logo icon here because it were render JS code here. We were already render the svg icon directly without any any <custom-element> like <icon-ia-logo> here.
So using the same SVG file and moved inside this repo to prevent future issues similar like this because we faced this  issue many times where the JS code was rendered instead of svg logo.